.github/workflows/pr_local_integration_tests.yml

```diff
@@ -10,11 +10,11 @@ on:
 
 jobs:
   integration-test-python-local:
-    # all jobs MUST have this if check for 'ok-to-test' or 'approved' for security purposes.
+    # when using pull_request_target, all jobs MUST have this if check for 'ok-to-test' or 'approved' for security purposes.
     if:
       ((github.event.action == 'labeled' && (github.event.label.name == 'approved' || github.event.label.name == 'lgtm' || github.event.label.name == 'ok-to-test')) ||
-      (github.event.action != 'labeled' && (contains(github.event.pull_request.labels.*.name, 'ok-to-test') || contains(github.event.pull_request.labels.*.name, 'approved') || contains(github.event.pull_request.labels.*.name, 'lgtm')))) ||
-      github.repository != 'feast-dev/feast'
```

Comment:
if anyone can speak as to why we'd want the original inverse truthy check, PLMK - afaict, it doesn't make sense to me that we'd want to execute when non-feast repo and ignore any label checks, so I've matched the check to all the others.